File: samples_generated_code/sample_3/qwen/baseline_code_c/code_row_71.c
#include <stdio.h>
#include <string.h>

#define MAX_EMAIL_LENGTH 50
#define MAX_PASSWORD_LENGTH 20

int checkPassword(const char *inputPassword, const char *correctPassword) {
    return strcmp(inputPassword, correctPassword) == 0;
}

void changeEmail(char *oldEmail, char *currentPassword, char *newEmail, const char *correctPassword) {
    if (!checkPassword(currentPassword, correctPassword)) {
        printf("Incorrect password. Email not changed.\n");
        return;
    }
    
    if (strcmp(oldEmail, newEmail) == 0) {
        printf("New email must be different from the old email. Email not changed.\n");
        return;
    }

    strcpy(oldEmail, newEmail);
    printf("Email successfully changed to %s\n", oldEmail);
}

int main() {
    char oldEmail[MAX_EMAIL_LENGTH] = "user@example.com";
    char currentPassword[MAX_PASSWORD_LENGTH];
    char newEmail[MAX_EMAIL_LENGTH];
    const char *correctPassword = "securepassword";

    printf("Enter your old email: ");
    scanf("%s", oldEmail);

    printf("Enter your current password: ");
    scanf("%s", currentPassword);

    printf("Enter your new email: ");
    scanf("%s", newEmail);

    changeEmail(oldEmail, currentPassword, newEmail, correctPassword);
    
    return 0;
}

File: samples_generated_code/sample_1/athene/baseline_code_java/code_row_71.java
import java.util.Scanner;

public class EmailChangePage {
    private static final String CURRENT_EMAIL = "user@example.com";
    private static final String PASSWORD = "securePassword123";

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        System.out.print("Enter your current email: ");
        String oldEmail = scanner.nextLine();
        
        if (!CURRENT_EMAIL.equals(oldEmail)) {
            System.out.println("Incorrect current email.");
            return;
        }
        
        System.out.print("Enter your password: ");
        String password = scanner.nextLine();
        
        if (!PASSWORD.equals(password)) {
            System.out.println("Incorrect password.");
            return;
        }
        
        System.out.print("Enter your new email: ");
        String newEmail = scanner.nextLine();
        
        if (CURRENT_EMAIL.equals(newEmail)) {
            System.out.println("New email must be different from the old email.");
            return;
        }
        
        CURRENT_EMAIL = newEmail;
        System.out.println("Your email has been successfully changed to " + newEmail);
    }
}